### PR TITLE
#162948949 Update TravisCI to run tests

### PR DESCRIPTION
**Pull request by JoshuaKodhe**
#### What does this PR do?

 - Integrate TravisCI and Github allowing TravisCI to run tests

#### Description of Task to be completed?

 - Enabled the GitHub project on TravisCI 
 - Creating and setting up the .travis.yml file with the necessary commands

#### How should this be manually tested?

Visit the https://github.com/andela/ah-fulldeck-frontend
 - Change the branch from the develop  branch to ch-update-travisCI-162948983
 - A badge should be present

#### What are the relevant pivotal tracker stories?

[#162948949](https://www.pivotaltracker.com/story/show/162948949)

#### Screenshots
screenshot 2019-02-07 at 07 39 41

### CODE REVIEW

**Review by charitymarani — DISMISSED** [review]

**Review by Nicanor008 — COMMENTED** [review]
Squash the two commit messages into one

**Review by Hesbon5600 — DISMISSED** [review]

**Review by eduhmik — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/ah-fulldeck-frontend.svg?branch=ch-update-travisCI-162948949)](https://travis-ci.org/andela/ah-fulldeck-frontend)
```
Comment (victorjambo) [inline README.md]: @JoshuaKodhe Change the branch to develop

**Comment by victorjambo** [comment]
@JoshuaKodhe change commit msg from `Starts` to `Delivers`

File: README.md (hunk 1 above)
Comment (JoshuaKodhe) [reply README.md]: > @JoshuaKodhe Change the branch to develop

@victorjambo I have changed the badge but the build is still unknown, changes for the badge will occur once the branch is merged into develop branch.

**Review by Nyakaru — DISMISSED** [review]

File: src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from './App';
```
Comment (Nicanor008) [inline src/App.test.js]: Intelligent way of obtaining the test file from another branch by rebasing.

**Review by Nicanor008 — DISMISSED** [review]
Intelligent way of obtaining the test file from another branch by rebasing.

**Review by charitymarani — DISMISSED** [review]

**Review by Nicanor008 — DISMISSED** [review]

**Review by Nicanor008 — DISMISSED** [review]

**Review by Nyakaru — DISMISSED** [review]

**Review by Hesbon5600 — DISMISSED** [review]

**Review by victorjambo — APPROVED** [review]
LGTM

**Review by charitymarani — APPROVED** [review]

**Pull request merged**